force-app/main/default/classes/accRecController.cls

```diff
@@ -2,13 +2,11 @@ public with sharing class accRecController
 { 
      public static Map<Id, String> recordtypemap {get;set;}
     	@AuraEnabled
-        public static string getAccountEmail(string accID)
-        {
+        public static string getAccountEmail(string accID){
```

Comment:
There was no need to remove the extra spaces. It also comes under best practices.

Reply:
Okay. For now I am putting as is and will remember from next time.

Reply:
Thanks, 

Please merge !!